src/utils/generateDirectionalStyles.js

```diff
@@ -17,22 +17,24 @@ function separateDirectionalStyles(originalStyles, autoRTLStyles) {
       }
 
       if (value && typeof value === 'object') {
+        delete ltrStyles[key];
```

Comment:
This is a patch for an unrelated issue where shared nested styles were being left on the ltr object and not being put on the shared object. I have written a test for this as well.